doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -331,36 +333,38 @@ set with:
 
 .. _bluestore-rocksdb-sharding:
 
-rocksdb sharding
+RocksDB Sharding
 ================
 
-internally bluestore uses multiple types of key-value data,
-stored in rocksdb.  each data type in bluestore is assigned a
-unique prefix. until pacific all key-value data was stored in
-single rocksdb column family: 'default'.  since pacific,
-bluestore can divide this data into multiple rocksdb column
-families. when keys have similar access frequency, modification
-frequency and lifetime, bluestore benefits from better caching
-and more precise compaction. this improves performance, and also
-requires less disk space during compaction, since each column
-family is smaller and can compact independent of others.
+BlueStore uses several types of internal key-value data, all of which are
+stored in RocksDB. Each data type in BlueStore is assigned a unique prefix.
+Prior to the Pacific release, all key-value data was stored in a single RocksDB
+column family: 'default'. In Pacific and later releases, however, BlueStore can
+divide key-value data into several RocksDB column families. BlueStore enjoys
```

Comment:
s/enjoys/achieves/

Reply:
Accepted. GitHub folded this one up, and I missed it. But then I saw it.